numpy/core/_add_newdocs.py

```diff
@@ -1354,8 +1354,7 @@
             decimal numbers, an operation which is better spelt
             ``frombuffer(string, dtype, count)``. If `string` contains unicode
             text, the binary mode of `fromstring` will first encode it into
-            bytes using either utf-8 (python 3) or the default encoding
-            (python 2), neither of which produce sane results.
+            bytes using utf-8, which will not produce sane results.
```

Comment:
Maybe we should just remove the kwarg `sep` since it has been deprecated for long enough

Reply:
The purpose of this merge request was to fix documentation only, not to modify actual code. I have nothing against broadening its scope, but please make sure you all agree on the new scope.